docs/compliance/nist/800_190.md

```diff
@@ -0,0 +1,9 @@
+---
+description: NIST SP 800-190 application container security guide
+keywords: standards, compliance, security
+title: NIST SP 800-190 application container security guide
+---
+
+In September 2017, NIST released [Special Publication (SP) 800-190](https://csrc.nist.gov/publications/detail/sp/800-190/final), Application Container Security Guide. SP 800-190 includes guidance and recommendations for the secure use of containers and incorporates elements of operating system and container runtime security.
+
+It is important to note that the publication merely serves as a guide and only calls out examples specific of the Docker container runtime where applicable. The publication is meant to help enhance your organizational security posture and support your risk management processes as they pertain to container usage.
```

Comment:
s/It is important to note that the/The

s/meant/intended... Or even "The intent of this publication is to help enhance..."

I don't like "organizational security posture". Maybe "practices"? "standards"?

Reply:
Will fix